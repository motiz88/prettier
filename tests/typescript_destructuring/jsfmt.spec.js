run_spec(__dirname, { parser: "typescript" });
