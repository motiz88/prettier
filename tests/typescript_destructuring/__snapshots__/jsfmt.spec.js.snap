// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`destructuring.ts 1`] = `
({ foo = [] } = bar);
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
({ foo = [] } = bar);

`;
